pkg/controller/provider/container/ova/model.go

```diff
@@ -242,7 +242,7 @@ func (r *StorageAdapter) GetUpdates(ctx *Context) (updates []Updater, err error)
 
 // List the collection.
 func (r *StorageAdapter) List(ctx *Context, provider *api.Provider) (itr fb.Iterator, err error) {
-	storageName := fmt.Sprintf("Dummy storage for %s", provider.Name)
+	storageName := fmt.Sprintf("Dummy storage for source provider %s", provider.Name)
```

Comment:
I'm not sure if the `provider` is needed, and maybe change to something like `dummy source provider storage <provider_name>`

Reply:
`Dummy storage for source provider %s` looks great